src/extensionProvider.ts

```diff
@@ -1,26 +1,25 @@
-import { IPlainTransactionObject } from "@multiversx/sdk-core/out";
-import { SignableMessage } from "@multiversx/sdk-core/out/signableMessage";
+import { IPlainTransactionObject, Message } from "@multiversx/sdk-core";
```

Comment:
`Address` can also be imported here instead on another separate import on line 8